### ISSUE TITLE
Pins on PCB Board are too small

### ISSUE THREAD

**Issue by mattwrdg** [roman-minyaylov/fritzing-parts#2]
Hi, 
first of all, I'd like to thank you very much for your effort. 

Unfortunately, I've ordered some PCB Boards with the Node MCU on it. To me, it seems the pins are designed to small. 
Because the NodeMCU with the normal Pins doesn't fit into the wholes. 
I've looked within the fritzing app if I can adjust this quickly, but I'm not that familiar with it. 
So to me, it would be nice if you could have a look at the pin size. E.g. the OLED display 0.96" shows thicker output pins.

Thanks and best regards,
Matt

**Comment by jamiehook** [comment]
Yup they are too small, I ordered a PCB, they are in the right place exactly, just not big enough

**Comment by mattwrdg** [comment]
Any news or ideas how to solve this issue ?

**Comment by jamiehook** [comment]
I sent a PR a while ago for this repo: https://github.com/roman-minyaylov/nodemcu-v3-fritzing should work there.

**Comment by roman-miniailov** [comment]
Hi guys. I can accept PR if somebody will resolve issue. It was many years ago when I publish this project and I don't remember how all worked, unfortunately.

**Comment by sofar** [comment]
This just cost me 65$.
